### DISCUSSION TITLE
Merge Unix timestamp and datetime formats in scrubberWrapper textbox

### DISCUSSION THREAD

**Discussion by t-yep-cpr** [foxglove/community#936] [Q&A]
At our organization, we use Unix timestamps very frequently. 
It would be nice if Unix timestamps can be included along with the default datetime format you have in the scrubberWrapper timestamp textbox that appears when you hover over the seeker bar. This would save users from having to:

- Find where exactly its located.
- Changing between the two timestamp formats. 

Combining the timestamp formats, I believe, will improve UX for roboticists as they continue to use your visualizer to perform troubleshooting.

![change_timestamp_foxglove](https://github.com/foxglove/community/assets/159342923/9eb9d96f-a6c8-405b-81d9-1b8c50e197b6)

**Comment by defunctzombie** [comment]
@t-yep-cpr Is the ask here to show the unix timestamp _and_ the human-readable timestamp in the scrubber popup? Whereas currently we only show one or the other based on your settings?

**Reply by t-yep-cpr** [reply]
Hi @defunctzombie , yes, that's what I was getting at.

**Reply by jtbandes** [reply]
Internal tracking issue: FG-6601
